src/internal/ppsutil/decoder.go

```diff
@@ -52,16 +51,6 @@ func NewPipelineManifestReader(path string) (result *PipelineManifestReader, ret
 			return nil, err
 		}
 	}
-	// TODO(msteffen): if we can get the yaml decoder to handle leading tabs, as
```

Comment:
The point of this code block is to determine whether we need to use `encoding/json` or `gopkg.in/pachyderm/yaml` to parse pipeline specs. Since the overall point of this PR is get rid of `encoding/json` at the cost of slight backwards incompatibility, this code block is no longer necessary.

The comment references the fact that `gopkg.in/yaml` can't parse JSON-style documents with leading tabs. This is a bug in the yaml parser, but it's one that I don't think will affect our users. (At the time, I didn't want to break existing pipeline specs with leading tabs, thus the comment, but since we're about to GA 2.0 anyway, and since realistically I think this hedging was more motivated by anxiety than user data, I don't think the comment is really relevant anymore and probably never was)